service/controller/v26/cloudconfig/master_template.go

```diff
@@ -213,8 +213,8 @@ Requires=k8s-kubelet.service
 After=k8s-kubelet.service
 
 [Service]
-Environment="KUBECONFIG=/etc/kubernetes/config/addons-kubeconfig.yml"
-Environment="KUBECTL=quay.io/giantswarm/docker-kubectl:e777d4eaf369d4dabc393c5da42121c2a725ea6a"
+Environment="KUBECONFIG=/etc/kubernetes/kubeconfig/addons.yaml"
+Environment="KUBECTL=quay.io/giantswarm/docker-kubectl:0df981ec7d2bd3c1191f75a0890f1089762febfb"
 ExecStart=/bin/sh -c '\
 	while [ "$(/usr/bin/docker run -e KUBECONFIG=${KUBECONFIG} --net=host --rm -v /etc/kubernetes:/etc/kubernetes $KUBECTL get cs | grep Healthy | wc -l)" -ne "3" ]; do sleep 1 && echo "Waiting for healthy k8s";done;sleep 30s; \
```

Comment:
Same with this line. Seems way too complicated for what it is actually trying to do.

Reply:
There should be a number of retries here (however high) to prevent us from hitting the same issue again.

Reply:
Yeah, I can use the same retry logic as is used in the inner statement to make sure it fails loudly instead of retrying forever.